docs/core/extensions/windows-service.md

```diff
@@ -117,7 +117,7 @@ To create the .NET Worker Service app as a Windows Service, it's recommended tha
 > An alternative publishing approach is to build the *\*.dll* (instead of an *\*.exe*), and when you install the published app using the Windows Service Control Manager you delegate to the .NET CLI and pass the DLL. For more information, see [.NET CLI: dotnet command](../tools/dotnet.md).
 >
 > ```powershell
-> sc.exe create ".NET Joke Service" binpath="C:\Path\To\dotnet.exe C:\Path\To\App.WindowsService.dll"
+> sc.exe create ".NET Joke Service" binpath= "C:\Path\To\dotnet.exe C:\Path\To\App.WindowsService.dll"
```

Comment:
Why'd you add this space?

Reply:
To fix #36520 - and the tool itself is documented with spacing like that. I think its CLI arg parser follows some legacy standard or something. Some work without the space.